### ISSUE TITLE
"Option" title is missing context

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#2699]
Options is a label used in three different contexts (filter, sorting and annotations). The first two ones have contextualised titles (Filter option, Sorting options) but the third is missing context and only announces "Option". It should be "Annotation options". 

Also thinking that the three could benefit from inversion (Options for filters, Options for Sorting, Options for annotations) so the important information (Option) appears before the context information.
